spring-cloud-dataflow-admin-starter/src/main/java/org/springframework/cloud/dataflow/admin/config/AdminConfiguration.java

```diff
@@ -158,9 +163,13 @@ public void configurePathMatch(PathMatchConfigurer configurer) {
 	}
 
 	@Bean
-	@ConditionalOnMissingBean
+	@ConditionalOnMissingBean(TapOnChannelRecoveryStrategy.class)
```

Comment:
The other completion related beans don't have this. Also, there is always the ability to override by name, I'm not sure I understand the rationale here...